### DISCUSSION TITLE
Literature

### DISCUSSION THREAD

**Discussion by federicoVisi** [federicoVisi/SoundS-gesture-sound-interaction#2] [General]
I added some Literature here: https://github.com/federicoVisi/SoundS-gesture-sound-interaction/tree/main/Literature

@genesis-victoria : since you mentioned Karen Barad this time, I also included a research paper on gesture-sound interaction that is informed by Barad's work: https://github.com/federicoVisi/SoundS-gesture-sound-interaction/blob/main/Literature/jarvis_vannort_MOCO18.pdf

@biancadagger : this is the paper on the performance inspired by Xenofeminism: https://github.com/federicoVisi/SoundS-gesture-sound-interaction/blob/main/Literature/ob-scene-a-live-audiovisual-performance-for-photoplethysmograph-and-female-body.pdf

**Comment by biancadagger** [comment]
Thank you for posting, Federico!

On Tue, Jul 5, 2022 at 2:49 PM Federico Visi ***@***.***>
wrote:

> I added some Literature here:
> https://github.com/federicoVisi/SoundS-gesture-sound-interaction/tree/main/Literature
>
> @genesis-victoria <https://github.com/genesis-victoria> : since you
> mentioned Karen Barad this time, I also included a research paper on
> gesture-sound interaction that is informed by Barad's work:
> https://github.com/federicoVisi/SoundS-gesture-sound-interaction/blob/main/Literature/jarvis_vannort_MOCO18.pdf
>
> @biancadagger <https://github.com/biancadagger> : this is the paper on
> the performance inspired by Xenofeminism:
> https://github.com/federicoVisi/SoundS-gesture-sound-interaction/blob/main/Literature/ob-scene-a-live-audiovisual-performance-for-photoplethysmograph-and-female-body.pdf
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/federicoVisi/SoundS-gesture-sound-interaction/discussions/2>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZBBLDQA6EFMLBKTJJUEBSLVSQVM7ANCNFSM52WDO6MA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
*Heather Fae *
heatherfae.com
